test/test_autograd.py

```diff
@@ -5556,7 +5559,10 @@ def bar(a):
 
     def test_jvp_err_check_strict(self):
         def foo(a):
-            return a.detach()
+            # break both forward and backward mode links
+            out = a.detach()
+            out, _ = fwAD.unpack_dual(out)
+            return out
```

Comment:
`test_jvp_err_check_strict` doesn't call `jvp` with `fw_mode=True`, so the unpack_dual line here doesn't actually do anything meaningful, right?

On a related note, it would be good to update all of the jvp tests to test with both `fw_mode=True` and `fw_mode=False` for comprehensiveness. I see the following functions:
- test_jvp_output
- test_jvp_scalar
- test_jvp_err_check
- test_jvp_err_check_strict
- test_jvp_create_graph (has already been updated to test fw_mode=True)